dev/create-release/spark-rm/Dockerfile

```diff
@@ -40,7 +40,9 @@ ARG APT_INSTALL="apt-get install --no-install-recommends -y"
 # TODO(SPARK-32407): Sphinx 3.1+ does not correctly index nested classes.
 #   See also https://github.com/sphinx-doc/sphinx/issues/7551.
 #   We should use the latest Sphinx version once this is fixed.
-ARG PIP_PKGS="sphinx==3.0.4 mkdocs==1.1.2 numpy==1.19.4 pydata_sphinx_theme==0.4.1 ipython==7.19.0 nbsphinx==0.8.0 numpydoc==1.1.0"
+# TODO(SPARK-35375): Jinja2 3.0.0+ causes error when building with Sphinx.
+#   See also https://issues.apache.org/jira/browse/SPARK-35375.
+ARG PIP_PKGS="sphinx==3.0.4 mkdocs==1.1.2 numpy==1.19.4 pydata_sphinx_theme==0.4.1 ipython==7.19.0 nbsphinx==0.8.0 numpydoc==1.1.0 jinja2==2.11.3"
```

Comment:
Why not also `jinja2<3.0.0`?

Reply:
Other packages are also fixed their version in this file, so should we fix the version for Jinja2 in this file too?

Reply:
Other dependencies are also fixed their version in this file, so should we fix for Jinja2 too?

Reply:
Oh okay. I'm just curious why it is not as same as the other place. If we know the version works, then it's fine.